chain/store/snapshot.go

```diff
@@ -123,11 +123,9 @@ func (cs *ChainStore) Import(ctx context.Context, r io.Reader) (*types.TipSet, e
 	}
 
 	ts := root
+	tssToPersist := make([]*types.TipSet, 0, uint64(TipsetkeyBackfillRange)*build.BlocksPerEpoch)
 	for i := 0; i < int(TipsetkeyBackfillRange); i++ {
-		err = cs.PersistTipset(ctx, ts)
-		if err != nil {
-			return nil, err
-		}
+		tssToPersist = append(tssToPersist, ts)
 		parentTsKey := ts.Parents()
 		ts, err = cs.LoadTipSet(ctx, parentTsKey)
```

Comment:
Sanity checking since I don't know this code: is this actually going to work now that we are not persisting tipsets one by one in this loop before the check?  We're putting these tipsets to cs somewhere else first right?

Reply:
My understanding is yes, because we've already `Put` all of these objects into our store above. This loop is about:
- sanity checking that we have them
- writing the TsKeys into the store for Eth stuff

It's probably not actually any faster to batch this, but I figured we might as well for cleanness.